usaspending_api/search/delta_models/subaward_search.py

```diff
@@ -362,8 +362,8 @@
         bs.award_description,
         bs.naics,
         bs.naics_description,
-        bs.cfda_numbers,
-        bs.cfda_titles,
```

Comment:
Might not be necessary, but I think it would help readability if you added `AS <old_column_name>` qualifiers on these three columns. This will help us be clear what the target column is on `subaward_search`.